reactor-core/src/main/java/reactor/core/publisher/Mono.java

```diff
@@ -3345,26 +3387,21 @@ public String toString() {
 			@Nullable Consumer<? super T> onNext,
 			@Nullable Consumer<? super Throwable> onError,
 			@Nullable Runnable onComplete,
-			@Nullable Runnable onAfterTerminate,
```

Comment:
I would have maybe opened a separate issue for that orthogonal change, :+1: for the change itself